.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 yarn.lock
```

Comment:
```suggestion

```

It is suggested to keep the yarn.lock file in your git, unless in some cases when it has yarn links to local projects. Idk you may look into it

Reply:
@sauravhiremath I don't know. I did not add that. Even I thought about it but I did not want to change it.